src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js

```diff
@@ -0,0 +1,45 @@
+import { loc, createCallout } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorVerifyFooter from '../../components/AuthenticatorVerifyFooter';
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    className: 'okta-verify-resend-push',
+
+    title () {
+      return loc('oie.okta_verify.push.title', 'login');
+    },
+
+    save () {
+      return loc('oie.okta_verify.push.send', 'login');
+    },
+
+    showMessages () {
+      // override showMessages to display error in cases like reject push
+      // or timeout. Borrowed this logic from TerminalView.
+      const messagesObjs = this.options.appState.get('messages');
+      if (messagesObjs && Array.isArray(messagesObjs.value)) {
+        this.add('<div class="ion-messages-container"></div>', '.o-form-error-container');
+
+        messagesObjs.value.forEach(messagesObj => {
+          const msg = messagesObj.message;
+          if (messagesObj.class && messagesObj.class === 'ERROR') {
```

Comment:
We use `optional chaining` now in SIW. This seems like a good candidate for it -
```
if (messagesObj?.class === 'ERROR') {
```
Same concept can be used in line 22 and line 29 (probably) here.

Reply:
You're right! took this from terminalView.. will change here 👍